src/axis.ts

```diff
@@ -203,12 +211,23 @@ export interface AxisConfig {
    * @TJS-type integer
    */
   titleMaxLength?: number;
-
+  /**
+   * The padding, in pixels, between title and axis.
+   */
+  titlePadding?: number;
   // ---------- Other ----------
   /**
    * Optional mark definitions for custom axis encoding.
    */
   encode?: VgAxisEncode;
+  /**
+   * The minimum extent (in pixels) that axis ticks and labels should use. This determines a minimum offset value for axis titles.
```

Comment:
I think the old description is quite convoluted. 

> The minimum extent (in pixels) that axis ticks and labels should use. This determines a minimum offset value for axis titles.

How about something like:

"Minimum extent, which determines the offset between axis ticks and labels.  "